src/event.js

```diff
@@ -573,7 +574,7 @@ jQuery.Event.prototype = {
 
 		this.isPropagationStopped = returnTrue;
 
-		if ( e ) {
+		if ( e && this.executeOriginalPropagation ) {
```

Comment:
Comments would have to be duplicated in two places.

This is my preference to put it simply, i would like to be explicit on what this is and why it was introduced